torch/_C/_autograd.pyi

```diff
@@ -91,7 +91,7 @@ def _profiler_enabled() -> bool: ...
 def _add_metadata_json(key: str, value: str) -> None: ...
 def _kineto_step() -> None: ...
 def kineto_available() -> bool: ...
-def _record_function_with_args_enter(name: str, args: List[Any]) -> torch.Tensor: ...
+def _record_function_with_args_enter(name: str, *args) -> torch.Tensor: ...
```

Comment:
How is this change relate to the rest of the PR? (Is it necessary?)

Reply:
This is required for the internal diff (which has other code depends on this external diff). This fixes an edge case where `args` is a single value, which would work correctly with `*args` but not `args: List[Any]`. A workaround is to put the single value in a list, but that would change its type. `List[Any]` is a mismatch of `py::args args` semantics (see https://pybind11.readthedocs.io/en/stable/advanced/functions.html#accepting-args-and-kwargs)